storage/src/main/java/zipkin2/storage/kafka/KafkaStorage.java

```diff
@@ -94,14 +100,18 @@
     this.spansTopicName = builder.spansTopicName;
     this.traceTopicName = builder.traceTopicName;
     this.dependencyTopicName = builder.dependencyTopicName;
-    // State store directories
-    this.storageDirectory = builder.storeDir;
+    // Storage directories
+    this.storageDir = builder.storageDir;
+    this.minTracesStored = builder.minTracesStored;
+    this.httpPort = builder.httpPort;
     // Kafka Configs
     this.adminConfig = builder.adminConfig;
     this.producerConfig = builder.producerConfig;
     this.aggregationStreamConfig = builder.aggregationStreamConfig;
     this.traceStoreStreamConfig = builder.traceStoreStreamConfig;
     this.dependencyStoreStreamConfig = builder.dependencyStoreStreamConfig;
+    this.prometheusRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+    Metrics.addRegistry(prometheusRegistry);
```

Comment:
do we need this or would be ok to reuse registry from server?

Reply:
Once using the same server, indeed you should just use the server's meterregistry - then all the metrics are on the same page.